src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt

```diff
@@ -39,6 +47,11 @@ class IntegrationTables {
          * 3. table_name
          * 4. start
          */
-        const val LOG_FAILED_INTEGRATION = "UPDATE integration_activity SET finish = now(), result = 'FAILED' WHERE integration_name = ? AND host_name = ? AND table_name = ? AND start = ? "
+        const val LOG_FAILED_INTEGRATION = "UPDATE integration_activity " +
+                "SET finish = now(), result = 'FAILED', cause = ? " +
```

Comment:
Won't this fail on existing `integration_activity` tables because of missing column? (not sure how launchpad we handle the table after integration finishes)

Reply:
I can just run a migration to add the column by hand once this gets merged

Reply:
Gotcha!